### PR TITLE
Fix bug closing categories directly after opening

### PR DESCRIPTION
**Pull request by Carlgo11**
There's a bug where a category table closes directly after being opened. To fix the bug I had to dissect the main.js file and rewrite the window.location.hash handling.

Notice the address bar in the following screen recording:

https://user-images.githubusercontent.com/3535780/164553655-54eddf53-197c-44c8-b5af-7aa507ba1ed1.mp4

### CODE REVIEW

**Comment by phallobst** [comment]
"Tell me you are paid per line without telling me you are paid per line."
```
// register service worker
registerServiceWorker();

// show region notice
showRegionNotice();
```
 :smile:

**Review by phallobst — APPROVED** [review]
I wasn't able to reproduce the bug with either the old or the new version. Oh well... fingers crossed!

**Review by dsoegijono — APPROVED** [review]

**Comment by phallobst** [comment]
@Carlgo11 The merge conflict needs to be resolved.

**Comment by Carlgo11** [comment]
Fixed the merge conflict. Just need a second review from someone before merging :wink:

**Review by phallobst — APPROVED** [review]

**Pull request merged**
